tests/unit/records/test_hints.py

```diff
@@ -17,7 +18,7 @@ def test_sniff_hints_from_fileobjs(self,
                                        mock_csv) -> None:
         mock_fileobj = MagicMock(name='fileobj')
         mock_fileobj.closed = False
-        mock_fileobjs = [mock_fileobj]
+        mock_fileobjs: List[IO[bytes]] = [mock_fileobj]
```

Comment:
Mypy was smart enough to complain that I was passing mock objects to a typed method.  Looks like it's happy if I tell it what type the value is pretending to be, though!